### DISCUSSION TITLE
11 colour wiz downlights successful 2 white-warm white not

### DISCUSSION THREAD

**Discussion by jazza1234567** [sbidy/wiz_light#115] [Q&A]
Hi all,

Been getting this up and running after a bit of teething, had no issues with the 11 RGB downlights I have, but for whatever reason the 2 W/WW downlights don't get recognised.

What can I do to contribute to getting these lights working for myself and anyone else :)

**Comment by sbidy** [comment]
Yes, this an bug for TW lights (Tunable White).
Can you please clone the current master branch for testing? Currently I don't have one of these bulb type available for testing.

**Reply by jazza1234567** [reply]
Thanks for getting back to me so quickly sbidy. 

Full disclosure, coding etc is not my strong suit so i may be a bit naive in knowing what to do. 

I have cloned the master yesterday to get the lights working. Ill go read through the info again to determine what I can do to test.

**Reply by jazza1234567** [reply]
@sbidy i posted a bulb test in the issues section. Hopefully its beneficial, if there's anything else i can do to help let me know. I'm more than happy to do whatever i can on my end
